python/iceberg/api/schema.py

```diff
@@ -80,6 +82,15 @@ def lazy_lowercase_name_to_id(self):
 
         return self._lowercase_name_to_id
 
+    def lazy_id_to_name(self):
+        from .types import index_by_name
```

Comment:
why the localised import?

Reply:
I haven't looked into it, but it's probably to prevent circular import.

Reply:
@jun-he can we add an issue to address python circular imports? I suspect its hiding an underlying structural issue in the python code.

Reply:
Issue is opened:
https://github.com/apache/iceberg/issues/2915

Reply:
Removed it.